src/node_api.h

```diff
@@ -166,6 +170,18 @@ NAPI_EXTERN napi_status napi_get_value_string_length(napi_env env,
                                                      napi_value value,
                                                      int* result);
 
+// Gets the number of BYTES in the LATIN-1 encoded representation of the string.
+NAPI_EXTERN napi_status napi_get_value_string_latin1_length(napi_env env,
```

Comment:
Taylor is removing `napi_get_value_string_utf8_length` and `napi_get_value_string_utf16_length` in https://github.com/nodejs/abi-stable-node/issues/193. In discussion, we decided it is cleaner to allow the `napi_get_value_string_*` APIs to optionally accept `nullptr` for the buffer and then return the length in bytes. So the same should be done for latin1.

We are keeping `napi_get_value_string_length` though, because the intention of that is to always get the *character* length.